app/models/server/models/_Base.d.ts

```diff
@@ -0,0 +1,19 @@
+export declare class Base {
+	model: any;
+
+	constructor(nameOrModel: string);
+
+	tryEnsureIndex(...args: any[]): any;
+
+	findOne(...args: any[]): any;
+
+	static findOne(...args: any[]): any;
+
+	insert(...args: any[]): any;
+
+	static insert(...args: any[]): any;
+
+	update(...args: any[]): any;
+
+	static update(...args: any[]): any;
```

Comment:
Why need these static methods?

Reply:
I honestly can't answer this right now, but I remember that this was needed in some cases due to the way current models are used